cmd/cometbft/commands/light.go

```diff
@@ -197,6 +197,9 @@ func runProxy(_ *cobra.Command, args []string) error {
 		)
 	}
 	if err != nil {
+		if errors.Is(err, light.ErrEmptyTrustedStore) {
+			logger.Error("Wanted to start from trusted store but it was empty. Please provide a trusted `--hash` and `--height` or check `--home-dir`.")
```

Comment:
Suggestion for the message.

"Starting the light client from an empty trusted store. Please provide an initialized trusted store, using the `---home-dir` flag, or provide trusted information to bootstrap the trusted store, via `--hash` and `--height` flags."